data/browsers/application.json

```diff
@@ -0,0 +1,1515 @@
+            "class": "VersionFactory",
+            "search": [
+                "FastBrowser"
+            ]
+        },
+        "engine": "webkit"
+    },
+    "flipboard app": {
+        "name": "Flipboard App",
+        "manufacturer": "Flipboard",
+        "pdfSupport": true,
+        "rssSupport": false,
+        "type": "application",
+        "version": {
+            "class": "VersionFactory",
+            "search": [
+                "Flipboard"
+            ]
+        },
+        "engine": "webkit"
+    },
+    "flixster app": {
+        "name": "Flixster App",
+        "manufacturer": "Flixster",
+        "pdfSupport": true,
+        "rssSupport": false,
+        "type": "application",
+        "version": {
+            "class": "VersionFactory",
+            "search": [
+                "FlixsteriOS"
+            ]
+        },
+        "engine": "webkit"
+    },
+    "frontpage": {
+        "name": "FrontPage",
+        "manufacturer": "Microsoft",
+        "pdfSupport": true,
+        "rssSupport": false,
+        "type": "application",
+        "version": {
+            "class": "\\BrowserDetector\\Version\\MicrosoftFrontPage",
+            "search": null
+        },
+        "engine": "unknown"
+    },
+    "google app": {
+        "name": "Google App",
+        "manufacturer": "Google",
+        "pdfSupport": true,
+        "rssSupport": false,
+        "type": "application",
+        "version": {
+            "class": "VersionFactory",
+            "search": [
+                "GSA"
+            ]
+        },
+        "engine": null
+    },
+    "google mobile app": {
+        "name": "Google Mobile App",
+        "manufacturer": "Google",
+        "pdfSupport": true,
+        "rssSupport": false,
+        "type": "application",
+        "version": {
+            "class": "VersionFactory",
+            "search": [
+                "GoogleMobile"
+            ]
+        },
+        "engine": null
+    },
+    "google+ app": {
+        "name": "Google+ App",
+        "manufacturer": "Google",
+        "pdfSupport": true,
+        "rssSupport": true,
+        "type": "application",
+        "version": {
+            "class": null,
+            "search": null
+        },
+        "engine": null
+    },
+    "ibooks author": {
+        "name": "iBooks Author",
+        "manufacturer": "Apple",
+        "pdfSupport": true,
+        "rssSupport": false,
+        "type": "application",
+        "version": {
+            "class": null,
+            "search": null
+        },
+        "engine": "webkit"
+    },
+    "instagram app": {
+        "name": "Instagram App",
+        "manufacturer": "Facebook",
+        "pdfSupport": true,
+        "rssSupport": false,
+        "type": "application",
+        "version": {
+            "class": "VersionFactory",
+            "search": [
+                "Instagram"
+            ]
+        },
+        "engine": "unknown"
+    },
+    "installatron": {
+        "name": "Installatron",
```

Comment:
__[Misspell]__

"Installatron" is a misspelling of "Installation"


<!-- Comment by [SideCI](https://sideci.com) -->